ember/app/controllers/contact.js

```diff
@@ -49,13 +49,13 @@ export default Ember.ObjectController.extend({
       });
 
       this.set("modalClosed", false);
-      Ember.$("body").addClass("modal-backing");
+      Ember.$(".main-container").addClass("modal-backing");
```

Comment:
Is this how you leverage jQuery within Ember? Sorry just trying to learn Ember from your code :)